WeakAuras/Transmission.lua

```diff
@@ -458,12 +469,12 @@ function StringToTable(inString, fromChat)
   return deserialized;
 end
 
-function WeakAuras.DisplayToString(id, forChat)
+function WeakAuras.DisplayToString(id, forChat, unique)
```

Comment:
Unique seems to be true for all callers. If so remove it.

Reply:
More groundwork for future iteration. If you don’t believe that there’s a use case for exporting a non-unique display, then I’d be happy to hear it.

Reply:
I believe there's no use case for that. And even if, then adding the parameter is easy.